keeper/msg_server.go

```diff
@@ -287,15 +287,19 @@ func (k *Keeper) claimExternalYieldIBC(ctx context.Context) error {
 		if err != nil {
 			return errors.Wrapf(err, "unable to claim yield for %s/%s", provider, channelId)
 		}
-		if !yield.IsPositive() {
+		retryAmount, err := k.GetRetryAmountAndRemove(ctx, provider, channelId)
+		if err != nil {
+			return errors.Wrapf(err, "unable to get and remove retry amount for %s/%s", provider, channelId)
+		}
+		if !yield.Add(retryAmount).IsPositive() {
```

Comment:
nit: If the yield is valid, defining a variable reduces the math additions by one
```suggestion
                accruedYield := yield.Add(retryAmount)
		if !accruedYield.IsPositive() {
```